unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/SkinFeature.cs

```diff
@@ -0,0 +1,11 @@
+﻿public class SkinFeature : IPlugin
+{
+    public SkinFeature(IBaseVariable<bool> isFeatureEnabled)
+    {
+        isFeatureEnabled.Set(true);
+    }
+    
+    public void Dispose()
+    {
+    }
+}
```

Comment:
I think that just having the plugin class like this defeats the purpose of having a `PluginSystem`. I assume this feature is not compatible with the `PluginSystem` because it involves modifying the core avatar renderer code, as opposed to have the functionality as a code add-on. That's ok, but in that case we can just use `DataStore.i.featureFlags.flags` to query the feature flag or something similar.

Reply:
done! using `DataStore.i.featureFlags.flags` directly from `AvatarEditorHUDController`